### ISSUE TITLE
Feature request: new admin commands

### ISSUE THREAD

**Issue by xF3d33** [xF3d33/UltimateTeams#10]
Add some admin commands:
### ~teamadmin disband %team%~

**Comment by Feryc-Projects** [comment]
Would be nice if they worked for offline players

**Comment by xF3d33** [comment]
> Would be nice if they worked for offline players

yeah the kick command will be also for offline players; as for disband it's already like this

**Comment by xF3d33** [comment]
> Would be nice if they worked for offline players

btw if you encounter new issues let me know
